bin/pre_compile

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# heroku post_compile script
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+pushd $DIR/..
+
+# heroku requires cffi to be in requirements.txt in order for libffi to be installed.
+# https://github.com/heroku/heroku-buildpack-python/blob/master/bin/steps/cryptography
+# to avoid making it a requirement for other build systems, we'll inject it now
+# into the requirements.txt file
+# remove pymssql, it doesn't compile on heroku
+grep -v '^pymssql' requirements_all_ds.txt >> requirements.txt
```

Comment:
The only concern here is that if in the future we will add some other library to requirements_all_ds that breaks Heroku deployment we won't know about it... :\